lib/compat/wordpress-6.7/class-gutenberg-rest-templates-controller-6-7.php

```diff
@@ -0,0 +1,205 @@
+<?php
```

Comment:
Can you clarify to me what's different in the endpoint compare trunk? It's not really clear because of how the code is extended.

Reply:
There are some `@core-merge` comments in this file (I updated them in the last commit). Every function should have a comment either at the top of the function or inline detailing the changes. Is that enough, or is there a better way to document this?